services/server/src/lib/availabilitySlot.ts

```diff
@@ -0,0 +1,33 @@
+import { availabilitySlots, interviews, lessons } from "@litespace/models";
+import { Knex } from "knex";
+import { isEmpty } from "lodash";
+
+export async function deleteSlots({
+  currentUserId,
+  ids,
+  tx,
+}: {
+  currentUserId: number;
+  ids: number[];
+  tx?: Knex.Transaction;
```

Comment:
Let's make the tx required to force the consumer to pass it. 

```suggestion
  tx: Knex.Transaction;
```